dbt/adapters/trino/connections.py

```diff
@@ -482,6 +482,8 @@ def add_query(self, sql, auto_begin=True, bindings=None, abridge_sql_log=False):
             if without_comments == "":
                 continue
 
+            individual_query = self._add_query_comment(individual_query)
```

Comment:
How is this query comment persisted in the database? Trino only has client_tags and object comments?

Reply:
[query-comment](https://docs.getdbt.com/reference/project-configs/query-comment) is a string added as a comment at the beginning (or [end](https://docs.getdbt.com/reference/project-configs/query-comment#append-a-custom-comment)) of each query.

e.g. when using the following config:
```yaml
query-comment: "executed by dbt"
```
The resulting queries will have a comment prepended like this:
```sql
/* executed by dbt */

select ...
```